src/Migration/Adapter/Database/MySqlSchemaAdapter.php

```diff
@@ -161,12 +169,28 @@ public function quote(?string $value): string
         return $this->pdo->quote($value);
     }
 
+    /**
+     * Quote array of values.
+     *
+     * @param array|string|null $values
+     *
+     * @return string[]
+     */
+    public function quoteArray($values)
+    {
+        return array_map(function ($value) {
```

Comment:
array_map is 4x slower then `foreach`. For performance reasons better use a foreach loop here.

Reply:
Ok